services/TransactionRecord.proto

```diff
@@ -46,4 +47,5 @@ message TransactionRecord {
     TransferList transferList = 10; // All hbar transfers as a result of this transaction, such as fees, or transfers performed by the transaction, or by a smart contract it calls, or by the creation of threshold records that it triggers.
     repeated TokenTransferList tokenTransferLists = 11; // All Token transfers as a result of this transaction
     ScheduleID scheduleRef = 12; // Reference to the scheduled transaction ID that this transaction record represent
+    CustomFeesCharged customFeesCharged = 13; // The custom fees that were charged, and must have been paid if the transaction status resolved to SUCCESS
```

Comment:
A little confused by the comment, is this saying that the field could be filled with "potential charges that were not paid" if the transaction was not successful?

Reply:
Yes, these are just the required fees that are charged.

If the payer cannot afford them, instead of `SUCCESS` we would get `INSUFFICIENT_PAYER_BALANCE_FOR_CUSTOM_FEE`.

Let me think about how to word this more clearly.

Reply:
Is it better to use 
```suggestion
    repeated CustomFeeCharged customFeesCharged = 13; // The custom fees that were charged, and must have been paid if the transaction status resolved to SUCCESS
```

with the wrapper, the generated java code to get the charged fees will be:

```java
  transactionRecord.getCustomFeesCharged().getCustomFeesChargedList()
```